lib/bolt/util.rb

```diff
@@ -212,6 +212,17 @@ def deep_merge(hash1, hash2)
         hash1.merge(hash2, &recursive_merge)
       end
 
+      def deep_merge!(hash1, hash2)
+        recursive_merge = proc do |_key, h1, h2|
+          if h1.is_a?(Hash) && h2.is_a?(Hash)
+            h1.merge(h2, &recursive_merge)
```

Comment:
Should nested hashes be merged in place too?

Reply:
Ah, yes, probably. I was initially uncertain if that would make sense (or a difference) but I think nested ones should be merged in place too.